core/src/cell.rs

```diff
@@ -31,7 +32,7 @@ impl BlockInfo {
 pub struct CellMeta {
     #[serde(skip)]
     pub cell_output: CellOutput,
-    pub out_point: CellOutPoint,
+    pub out_point: OutPoint,
     pub block_info: Option<BlockInfo>,
     pub cellbase: bool,
```

Comment:
There is another struct `TransactionInfo` https://github.com/nervosnetwork/ckb/blob/develop/core/src/extras.rs#L19
It's similar to the `BlockInfo`, it seems that we can remove the `BlockInfo`, and we can remove the `pub cellbase: bool` as well, I think whether it's a `cellbase`  depends on the position in the Block. https://github.com/nervosnetwork/ckb/blob/develop/store/src/store.rs#L195

Reply:
Maybe create a new issue for another PR.

Reply:
https://github.com/nervosnetwork/ckb/pull/1361